src/test/util_string_tests.cpp

```diff
@@ -122,6 +122,12 @@ BOOST_AUTO_TEST_CASE(ConstevalFormatString_NumSpec)
     FailFmtWithError<2>("%1$*2$", err_term);
     FailFmtWithError<2>("%1$.*2$", err_term);
     FailFmtWithError<2>("%1$9.*2$", err_term);
+
+    // Non-parity between tinyformat and ConstevalFormatString
```

Comment:
In commit "test: Document non-parity between tinyformat and ConstevalFormatstring for "%n"" (2c82ee3219ee186b9d3998a129bff1bc809894d3):

Another case where this is not parity is when a noninteger type is used as a width like `strprintf("%*s", "hi", "hi"),` which throws a `tinyformat::format_error` exception "tinyformat: Cannot convert from argument type to integer for use as variable width or precision". A test could be added for this as well.